flixel/FlxCamera.hx

```diff
@@ -653,6 +665,8 @@ class FlxCamera extends FlxBasic
 	{
 		super();
 		
+		pixelPerfectRender = (FlxG.renderBlit);
```

Comment:
Also, there's no reason to move this to the constructor. Instance variable initializations already happen there.